letsencrypt/cli.py

```diff
@@ -693,6 +706,94 @@ def install(args, config, plugins):
     le_client.enhance_config(domains, config)
 
 
+def renew(args, cli_config, plugins):
+    """Renew previously-obtained certificates."""
+    cli_config = configuration.RenewerConfiguration(cli_config)
+    configs_dir = cli_config.renewal_configs_dir
+    for renewal_file in reversed(os.listdir(configs_dir)):
```

Comment:
Rather than being inline, this loop should probably call subroutines, one to handle the config merge and (if it's more than ~5 LOC) one to invoke `obtain_cert` for that config.